e2e-test/app_helper.py

```diff
@@ -35,32 +35,41 @@ def poll_for_app_url(workspaceId, app_type, proxy_url_name, azure_token, leo_url
 
     # prevent infinite loop
     poll_count = 20 # 30s x 20 = 10 min
```

Comment:
i know this is the existing variable name, but maybe rename to `polling_attempts_remaining`?